### ISSUE TITLE
Unify `Layout/SpaceInsideArrayLiteralBracket` & `Layout/SpaceInsideHashLiteralBraces` `EnforcedStyleForEmptyBraces/Brackets` detection and message style

### ISSUE THREAD

**Issue by garettarrowood** [rubocop-hq/rubocop#5304]
## Expected behavior

That `Layout/SpaceInsideArrayLiteralBracket` & `Layout/SpaceInsideHashLiteralBraces` apply the same rules and message style for `EnforcedStyleForEmptyBrackets`/`EnforcedStyleForEmptyBraces`.

## Actual behavior

With `EnforcedStyleForEmpty` of `no_space` specified for both cops:

```ruby
#  Layout/SpaceInsideArrayLiteralBrackets
C: Layout/SpaceInsideArrayLiteralBrackets: Do not use space inside empty array brackets.
a = [ ]
    ^^^
C: Layout/SpaceInsideArrayLiteralBrackets: Do not use space inside empty array brackets.
a = [    ]
    ^^^^^^

# Layout/SpaceInsideHashLiteralBraces
C: Layout/SpaceInsideHashLiteralBraces: Space inside empty hash literal braces detected.
a = { }
     ^

C: Layout/SpaceInsideHashLiteralBraces: Space inside empty hash literal braces detected.
d = {   }
     ^^^
```

With `EnforcedStyleForEmptyBraces` of `space` specified for both cops:

```ruby
#  Layout/SpaceInsideArrayLiteralBrackets
C: Layout/SpaceInsideArrayLiteralBrackets: Use one space inside empty array brackets.
c = []
    ^^

C: Layout/SpaceInsideArrayLiteralBrackets: Use one space inside empty array brackets.
d = [   ]
    ^^^^^

# Layout/SpaceInsideHashLiteralBraces
C: Layout/SpaceInsideHashLiteralBraces: Space inside empty hash literal braces missing.
a = {}
    ^

# No offense
b = {   }
```

Differences for `no_space`:
1. Array cop uses more `^` than Hash cop.
2. Array cop message is an imperative statement. Hash cop message is a passive statement.

Differences for `space`:
1. Array cop uses more `^` than Hash cop.  _Preference:_ Array style
2. Array cop message is an imperative statement. Hash cop message is a passive statement.
3. EmptyLiteral with more than one space is not an offense for Hash cop. _Preference:_ This should be an offense.

## Steps to reproduce the problem

Run the snippets above with the two different `EnforcedStyleForEmptyBraces` settings.

## RuboCop version

```
0.52.0 (using Parser 2.4.0.2, running on ruby 2.3.3 x86_64-darwin15)
```
